content/en/methods/timelines.md

```diff
@@ -299,6 +299,58 @@ Invalid or missing Authorization header.
 
 ---
 
+## View link timeline {#link}
+
+```http
+GET /api/v1/timelines/link?url=:url HTTP/1.1
+```
+
+View public statuses containing a link to the specified URL. This only lists statuses from people who have opted in discoverability features.
+
+**Returns:** Array of [Status]({{<relref "entities/status">}})\
+**OAuth:** Public. Requires app token + `read:statuses` if the instance has disabled public preview.\
+**Version history:**\
+4.3.0 - added
+
+#### Request
+
+##### Headers
+
+Authorization
+: Provide this header with `Bearer <user token>` to gain authorized access to this API method.
+
+##### Query parameters
+
+url
+: {{<required>}} String. The name of the hashtag (not including the # symbol).
+
+max_id
+: String. All results returned will be lesser than this ID. In effect, sets an upper bound on results.
+
+since_id
+: String. All results returned will be greater than this ID. In effect, sets a lower bound on results.
+
+min_id
+: String. Returns results immediately newer than this ID. In effect, sets a cursor at this ID and paginates forward.
+
+limit
+: Integer. Maximum number of results to return. Defaults to 20 statuses. Max 40 statuses.
+
+#### Response
+##### 200: OK
+
+##### 404: Not found
+
+The provided URL is not currently trending.
```

Comment:
this references trending. Does this mean that the URL provided needs to be one that is currently trending or the call will fail? We maybe should clarify this in the description of the endpoint, if this is the purpose and use of the API.

Reply:
This indeed only works for currently trending links. Edited the description accordingly.